app/subscriber/src/features/my-reports/view/ReportEdit.tsx

```diff
@@ -123,118 +110,6 @@ export const ReportEdit: React.FC = () => {
     } catch {}
   });
 
-  const handleExport = React.useCallback(
```

Comment:
Moved all this logic to a separate component ReportEditForm.  This new component is the report editor with the tabs.